sympy/physics/units.py

```diff
@@ -270,7 +270,7 @@ def free_symbols(self):
 amu = amus = gram / avogadro / mol
 mmu = mmus = gram / mol
 quart = quarts = Rational(231, 4) * inch**3
-eV = 1.602176487e-19 * J
+eV = 1.6021766208e-19 * J
```

Comment:
I totally forgot about that. To keep the precision as given, use `Float('1.6021766208','')*10**-19`; this will give a Float as close as possible to the digits given.